components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/util/OAuth2Util.java

```diff
@@ -1220,21 +1220,6 @@ public static String getOAuth2JWKSPageUrl(String tenantDomain) throws URISyntaxE
             return auth2JWKSPageUrl;
         }
 
-        /**
-         * This method is used to get the resolved URL for the JWKS Page.
-         *
-         * @return String of the resolved URL for the JWKS page.
-         * @throws URISyntaxException URI Syntax Exception.
-         */
-        public static String getOAuth2JWKSPageUrl() throws URISyntaxException {
```

Comment:
Since this is a public method wondering whether we should first deprecate this and then remove it in the future.

Reply:
Yes, +1.
Let's deprecate and remove with a version bump

Reply:
This method was introduced as an overloaded method for getOAuth2JWKSPageUrl(String tenantDomain) [1] which is being used widely.
The purpose of introducing getOAuth2JWKSPageUrl() with [2]  was to consume the IdentityUtil.resolveURL API which has been now removed with [3].
Since the usages of this method are now set to consume the old [1] method with [4], I thought it would be safe to remove this method. Since we may need to introduce framework changes with the feature, "tenant-qualifying endpoints", we would need to remove this method soon since we wouldn't be able to bump the framework version in this module to incorporate those changes if this method is kept.


[1] - https://github.com/wso2-extensions/identity-inbound-auth-oauth/blob/92711d34215089fd6cf0eb376f9f39a0fec77cac/components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/util/OAuth2Util.java#L1211

[2] - https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1345

[3] - https://github.com/wso2/carbon-identity-framework/pull/2865

[4] - https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1353

Reply:
But there be any outside extensions that could be depending on this. So IMO deprecating 1st would be best. 
@mefarazath WDYT?